src/components/fileUpload.js

```diff
@@ -322,12 +498,129 @@
           ],
         },
       },
-      fileList: {
+      listView: {
+        display: 'flex',
+        alignItems: 'center',
+        justifyContent: 'space-between',
+      },
+      gridView: {
+        display: 'flex',
+      },
+      gridItem: {
+        display: 'flex',
+        borderRadius: '0.3rem',
+        flexDirection: 'column',
+        border: '1px solid #eee',
+        marginRight: '0.9375rem',
+        marginBottom: '0.9375rem',
```

Comment:
Might as well make it `1rem` I guess? As I know Thomas always works with spacing of 2, 4, 8, 12 or 16px.